css/acquia_lift.navbar.css

```diff
@@ -96,68 +109,81 @@
   background-color: gray;
 }
 
-/* Nested menu padding */
-#navbar-administration .navbar-tray-horizontal .menu ul li {
-  padding-bottom: .3em;
-}
-
 /* Campaign drop-down */
-#navbar-administration .acquia-lift-menu-item .acquia-lift-campaign {
+#navbar-administration .navbar-tray-horizontal .acquia-lift-menu-item {
+  margin-top: .7em;
+  margin-bottom: .7em;
+  padding-top: 0;
+  padding-bottom: 0;
+}
+#navbar-administration .navbar-tray-horizontal .acquia-lift-menu-item .acquia-lift-campaign,
+#navbar-administration .navbar-tray-horizontal .acquia-lift-menu-item .acquia-lift-preview-option {
+  display: block;
+  width: 100%;
   float: left;
 }
-#navbar-administration .acquia-lift-menu-item .acquia-lift-campaign-edit {
+#navbar-administration .navbar-tray-horizontal .acquia-lift-menu-item .acquia-lift-campaign-edit,
+#navbar-administration .navbar-tray-horizontal .acquia-lift-menu-item .acquia-lift-variation-delete,
+#navbar-administration .navbar-tray-horizontal .acquia-lift-menu-item .acquia-lift-variation-rename {
+  position: relative;
+  display: block;
   float: right;
-  margin-left: 1.667em;
+  text-align: right;
 }
-
```

Comment:
Gotcha. I will roll them in and see how they affect some of the layout rework I did and adjust accordingly if needed.